readme.md

```diff
@@ -28,14 +28,113 @@ This is the draft of some future version of the CCS specification.
-- Added `post_comment` capability
-- Added `simplified_judgement_type_id` to `judgement` and `judgement_type` endpoints
+**Functional Changes**  
+   1. Add support for **post\_comment** capability (posting commentary)  
+   2. contests EP  
+      1. Change **penalty\_time** from integer minutes to **RELTIME** (breaking change)  
+   3. judgement-types EP  
+      1. Add property **simplified\_judgement\_type\_id** (ID ?)  
+   4. problems EP  
+      1. Add property **memory\_limit** (integer)  
+      2. Add property **output\_limit** (integer)  
+      3. Add property **code\_limit**  (integer)  
+      4. Add property **attachments** (array of FILE ?)  
+   5. organizations EP  
+      1. Add property **country\_subdivision** (string ?)  
+      2. Add property **country\_subdivision\_flag** (array of FILE ?)  
+   6. submissions EP  
+      1. Add property **account\_id** (ID ?)  
+      2. Made **team\_id** optional (ID ?)  
+   7. judgements EP  
+      1. Add property **current** (boolean ?)  
+      2. Add property **simplified\_judgement\_type\_id** (ID ?)  
+   8. clarifications EP  
+      1. Teams must have the **post\_clar** capability to post a clarification (previously was **team\_clar**)  (breaking change)  
+      2. Rule change about what properties may be specified in a **POST** when the **post\_clar** capability is present. (breaking change)  
+      3. Remove **to\_team\_id** and replace it with **to\_team\_ids** and **to\_group\_ids** to allow broader targeting of destination team(s). (breaking change)  
+   9. awards EP  
+      1. Added new rules for honors, high-honors and highest-honors  
+   10. commentary EP  
+       1. Add ability to modify (post) a comment  
+   11. scoreboard EP  
+       1. Change **score.total\_time** from **integer** to **RELTIME** (breaking change)  
+       2. Change **score.time** from **integer** to **RELTIME ?** (breaking change)  
+       3. Change **problems.time** from **integer** to **RELTIME** (breaking change)  
+   12. **Note:** Event feed notifications containing the above objects must also be updated as indicated.
+
+**Non-functional Changes** (documentation)  
+   1. Entire Document  
+      1. Change all endpoint URLs to be correctly relative by removing prefix slashes.  This applies to all endpoints in the document.  
+      2. Change [IETF.org](http://IETF.org) RFC references to use current website.  
+   2. General design principles  
+      1. Endpoint URLs  
+         1) Clarify that **baseurl** must end in a slash.  
+         2) Add examples and reference to RFC  
+      2. JSON property types  
+         1) Description of *Identifiers* clarified.  
+         2) Clarify no meaning is implied by the order of objects in arrays.  
+         3) Added note to href  
+         4) Added property **tag** (array of string) to file reference objects (FILE) along with a description of how it may be used. (eg. hints such as light or dark for images)  
```

Comment:
Nitpick: I'd say this is a functional change as it changes every endpoint that has a property of FILE type.